packages/messaging/lib/index.js

```diff
@@ -397,10 +363,8 @@ export default createModuleNamespace({
   nativeModuleName,
   nativeEvents: [
     'messaging_token_refresh',
-    'messaging_message_sent',
     'messaging_message_deleted',
```

Comment:
As above, this is part of the upstream messaging API which we're removing, given that the user can no longer `sendMessage` this will never trigger, so can be removed.